src/backend/marsha/core/templates/core/lti.html

```diff
@@ -9,7 +9,7 @@
   </head>
   <body>
     <div class="marsha-frontend-data"{% if jwt_token %} data-jwt="{{ jwt_token }}"{% endif %} data-state="{{ state }}"></div>
-    <div class="marsha-frontend-data" id="video" data-video="{{ video_data }}"></div>
+    <div class="marsha-frontend-data" id="{{ resource }}" data-{{ resource }}="{{ resource_data }}"></div>
```

Comment:
I would not change the name of the attribute...
```suggestion
    <div class="marsha-frontend-data" id="{{ resource }}" data-resource="{{ resource_data }}"></div>
```

Reply:
For now I would like to keep it like this. When I will refactor the front application maybe I would change with a generic `data-resource`.